website/pages/docs/plugins/sources/alicloud/tables.md

```diff
@@ -2,8 +2,8 @@
 
 ## Tables
 
-- [alicloud_bss_bill](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/alicloud/docs/tables/alicloud_bss_bill.md)
-- [alicloud_bss_bill_overview](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/alicloud/docs/tables/alicloud_bss_bill_overview.md)
-- [alicloud_ecs_instances](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/alicloud/docs/tables/alicloud_ecs_instances.md)
-- [alicloud_oss_buckets](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/alicloud/docs/tables/alicloud_oss_buckets.md)
-  - [alicloud_oss_bucket_stats](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/alicloud/docs/tables/alicloud_oss_bucket_stats.md)
\ No newline at end of file
+- [alicloud_bss_bill](/docs/plugins/sources/alicloud/tables/alicloud_bss_bill)
```

Comment:
I wonder if it should be absolute path (should the path be prefixed with `/`)?

Reply:
Good point, I made those relative